theseus/labs/lie/lie_tensor.py

```diff
@@ -55,15 +56,30 @@ def euclidean_enabled() -> bool:
 
 
 class _LieTensorBase(torch.Tensor):
-    def __new__(cls, data: torch.Tensor, ltype: _ltype, requires_grad=False):
-        return cls._make_subclass(cls, data, requires_grad)  # type: ignore
+    def __new__(
```

Comment:
Flagging @rmurai0610's comment here so we can maybe address it in this PR.

> Thank you Luis, it looks great!
> 
> I have a minor comment.
> For `LieTensor.new()`, should it share storage? 
> For example, `tensor.new_tensor` always copies the data, and I think the term new sort of implies that, and I think it would be nicer if we kept consistent.
> 
> Also, I'm not sure how exactly I should read this line.
> `x.new(data)` isn't very intuitive, maybe something like `x.new_lietensor` is more clear (similar to new_tensor)

Reply:
Fixed. cc @rmurai0610